### PR TITLE
Remove non-go option

### PR DESCRIPTION
**Pull request by adambabik**
`non-go` option is not necessary in dep `prune` section.

With `non-go` option enabled, there is a warning:
```
$ make dep-ensure
dep: WARNING: redundant prune option "non-go" set for "github.com/karalabe/hid"
dep: WARNING: redundant prune option "non-go" set for "github.com/ethereum/go-ethereum"
```

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -6,12 +6,10 @@ ignored = ["github.com/ethereum/go-ethereum/whisper/notifications"]
 
   [[prune.project]]
     name = "github.com/karalabe/hid"
-    non-go = false
```
Comment (dshulyak) [inline Gopkg.toml]: maybe it would be better to add non-go=true as a default for other packages and keep false for hid and geth?

Comment (adambabik) [reply Gopkg.toml]: That seems good. I will test it.

Comment (mandrigin) [reply Gopkg.toml]: Yeah, I thought it is true by default. But let's be explicit.

**Review by mandrigin — CHANGES_REQUESTED** [review]
Let's try setting `non-go=true` globally.

**Comment by mandrigin** [comment]
@adambabik now vendor check fails :(

**Comment by adambabik** [comment]
@mandrigin it's fixed. I needed to rebase as some artifacts left in `vendor/` from develop.

**Review by mandrigin — APPROVED** [review]

**Pull request merged**